backend/app/services/search.py

```diff
@@ -349,7 +349,10 @@ async def buildFilterData(self, filter, total):
                     platformOptions = buildPlatformFilter(upstreamList, clusterTypeList)
                     values = values + platformOptions
                 elif key == "ocpVersion":
-                    short_versions = [str(value)[0:6] for value in values]
+                    short_versions = [
+                        str(value)[0 : constants.OCP_SHORT_VERSION_LENGTH]
```

Comment:
I suspect it won't improve the wrapping here, but `[0:<n>]` has the same effect as just `[:<n>]`, which is a little shorter. And I don't think the formatter really wants the spaces here, either. `str(value)[:constants.OCP_SHORT_VERSION_LENGTH]` would work fine. (It's actually a bit funny that "short version length" is so long. You could probably try `OCP_SHORT_VER_LEN`, which *might* get this on one line.)

Reply:
Right! updated it